src/directives/apMesa.js

```diff
@@ -255,7 +255,12 @@
       scope.$watch('persistentState.sortOrder', function(sortOrder) {
         if (sortOrder) {
           scope.sortDirection = {};
-          sortOrder.forEach(function(sortItem) {
+          scope.sortPriority = {};
+          scope.sortPriorityShow = sortOrder.length > 1;
```

Comment:
instead of adding these directly to the scope, add them to `scope.transientState`